tests/functional/configs/test_warn_error_options.py

```diff
@@ -53,9 +53,8 @@ def test_can_silence(self, project, catcher: EventCatcher, runner: dbtRunner) ->
         assert_deprecation_warning(result, catcher)
 
```

Comment:
can't comment directly on the line, but is it clear why `TestWarnErrorOptionsFromProject` was passing prior to this fix?